.github/workflows/update-prerun-script.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         id: create-pull-request
         uses: peter-evans/create-pull-request@f094b77505fb89581e68a1163fbd2fffece39da1
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.JS_SDK_BOT_ACCESS_KEY }}
```

Comment:
Could you please help me understand: what is the difference between the two keys?

Reply:
The GITHUB_TOKEN uses the existing credentials whereas the JS_SDK_BOT_ACCESS_KEY is the PAT for the chime-js-sdk-bot account

Reply:
A PAT is a auth token for a "machine" user that has write access to our repo. We use the token to create a PR, so that the PR can trigger the github action jobs.